build-system/release-tagger/make-release.js

```diff
@@ -178,8 +178,12 @@ function _createBody(head, base, prs) {
     ? ''
     : dedent`\
     <h2>🌸 Cherry-picked release 🌸</h2>
-    <a href="https://github.com/ampproject/amphtml/releases/tag/${base}">\
-    ${base}</a> was patched and published as <b>${head}</b>. Refer to the \
+    <a href="https://github.com/ampproject/amphtml/releases/tag/${
+      head.slice(0, -3) + '000'
+    }">\
+    ${
+      head.slice(0, -3) + '000'
+    }</a> was patched and published as <b>${head}</b>. Refer to the \
```

Comment:
For readability and reusability, can we move `${head.slice(0, -3) + '000'}` into its own variable?